Moda.Web/src/moda.web.reactclient/app/services/clients.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+import { ProfileClient } from "./moda-api";
+import { acquireToken } from "./auth";
+
+
+export const getProfileClient = async () => new ProfileClient('', axios.create({
```

Comment:
I don't really like this, but I could not get axios interceptors to work to add the header. I think because it's an async call to acquireToken.